Sources/swift-bootstrap/main.swift

```diff
@@ -277,10 +277,13 @@ struct SwiftBootstrapBuildTool: ParsableCommand {
                 targetTriple: self.targetToolchain.targetTriple,
                 flags: buildFlags,
                 architectures: architectures,
-                useIntegratedSwiftDriver: useIntegratedSwiftDriver,
                 isXcodeBuildSystemEnabled: buildSystem == .xcode,
-                explicitTargetDependencyImportCheckingMode: explicitTargetDependencyImportCheck == .error ? .error : .none,
-                verboseOutput: logLevel <= .info
+                verboseOutput: logLevel <= .info,
```

Comment:
I think we should keep `verboseOutput` as the last argument

Reply:
WDYT of moving it and `colorizedOutput` to separate `outputParameters`, similar to other new structs introduced in this PR?